wallet/wallet.sol

```diff
@@ -363,16 +363,57 @@ contract Wallet is multisig, multiowned, daylimit {
             return true;
         }
     }
-    
+
+    // Gets the number of pending transactions
+    function numPendingTransactions() constant returns (uint) {
+        var pendingTransactionsCount = 0;
+        // Use m_pendingIndex.length to get all hashes, then count how many of the
+        // operations are transactions, because we don't want to store hashes twice
+        // and this is a local call anyway
+        for (uint i = 0; i < m_pendingIndex.length; i++) {
+            if (isPendingTransaction(i)) {
+              pendingTransactionsCount++;
+            }
+        }
+
+        return pendingTransactionsCount;
+    }
+
+    // Gets the pending operation at a specified index
+    // Will return a tuple [bytes32 operationHash, uint confirmationsNeeded, address toAddress, uint transactionValue, bytes data]
+    function getPendingTransaction(uint _index) constant returns (bytes32, uint, address, uint, bytes) {
```

Comment:
I think it would be better readable if you named the return parameters. Then it is also much harder to make a mistake an the actual return instruction.